marketpulse/main/templates/activities.html

```diff
@@ -23,8 +23,9 @@ <h4 class="text-center">
 
   <div class="row">
     <div class="col-md-12 text-right">
-      <a href="{{ url('main:user_fxosprice', user.pk) }}" class="view-all">View your contributions</a> &bull;
-      <a href="{{ url('main:all_fxosprice') }}" class="view-all">View all contributions</a>
+      <a href="{{ url('main:user_contributions') }}"
+         class="view-all">View your contributions</a> &bull;
```

Comment:
I prefer a limit of 100 chars in html files. This line limit makes it OK to see all the code in half split window. Speaking of other lines in the codebase and 100 limit per line, there is another one that we could break :P